packages/expo-device/src/ExpoDevice.web.ts

```diff
@@ -25,7 +25,15 @@ export default {
   get deviceYearClass(): null {
     return null;
   },
-  get totalMemory(): null {
+  get totalMemory(): number | null {
+    if ('deviceMemory' in navigator) {
+      // @ts-ignore
```

Comment:
Prefer `@ts-expect-error` so that we are aware we need to change the comment when the type declarations change. And always explain why the TS error is being suppressed in the comment.

Reply:
> Unused '@ts-expect-error' directive.ts(2578)

Reply:
If it's unused, then we should be able to remove `@ts-ignore` as well.